Snowflake.Data/Core/Session/SessionPool.cs

```diff
@@ -163,6 +175,8 @@ private Task<SFSession> NewSessionAsync(String connectionString, SecureString pa
         {
             s_logger.Debug("SessionPool::NewSessionAsync");
             var session = s_sessionFactory.NewSession(connectionString, password);
+            if (_pooling)
+                _busySessionsCounter.Increase();
```

Comment:
lets here move this counter increase to the ContinueWith phase, makes sense?

Reply:
done